src/sync/exercises.nim

```diff
@@ -68,7 +69,7 @@ proc findExercises*(conf: Conf): seq[Exercise] =
   let probSpecsExercises = findProbSpecsExercises(conf).mapIt((it.slug, it)).toTable
 
   for practiceExercise in findPracticeExercises(conf):
-    result.add(initExercise(practiceExercise, probSpecsExercises.getOrDefault(practiceExercise.slug)))
+    result.add(initExercise(practiceExercise, probSpecsExercises.getOrDefault(practiceExercise.slug.string)))
```

Comment:
Most of the things that use `.string` should be refactored away entirely, or become a `{.borrow.}`